style.css

```diff
@@ -16,9 +16,16 @@ body {
    background-color: #F5FFFA;
 }
 
+div.heading {
+   padding: 0 20px 20px 40px;
+   /*background-size: 30px;*/
+   background-color: #ADD8E6;
+   margin-bottom: 50px;
+}
+
 h1 {
    float: left;
-	color: #00E5EE;
+	color: #1E90FF;
```

Comment:
Check your indentation. Keeping an eye on it will save you hours in the long run.